source/locale/en/symbols.dic

```diff
@@ -172,6 +230,9 @@ _	line	most
 ⊇	superset of or equal to	none
 
 # Vulgur Fractions U+2150 to U+215E
+¼	one fourth	none
+½	one half	none
+¾	three fourths	none

```

Comment:
I believe "one quarter" and "three quarters" is much more popular than the also acceptable "fourth" version. I propose this be changed. 
https://english.stackexchange.com/questions/103188/three-quarters-vs-three-fourths